polaris-react/src/components/IndexTable/tests/IndexTable.test.tsx

```diff
@@ -382,6 +382,26 @@ describe('<IndexTable>', () => {
       );
     });
 
+    it('renders a custom pagination string if present', () => {
```

Comment:
```suggestion
    it('renders a custom select all string if present', () => {
```

The terminology is a little odd. The button doesn't actually handle any pagination. It just selects "all" resources